src/test/java/it/gov/pagopa/pu/workflow/event/payments/consumer/PaymentsConsumerTest.java

```diff
@@ -56,6 +84,8 @@ void givenExpectedEventWhenAcceptThenInvokeClient() {
       .startTransferClassification(new TransferClassificationStartSignalDTO(1L, "iuv3", "iur2", 1));
     Mockito.verify(wfClientMock)
       .startTransferClassification(new TransferClassificationStartSignalDTO(1L, "iuv5", "iur1", 1));
+    Mockito.verify(createAssessmentsWFClientMock, Mockito.times(3))
```

Comment:
3 times is not correct... I don't remember if a receipt could have more than 1 installment... let's change from List to Set the receiptId to collect